src/action-creators/__tests__/cursorPrev.js

```diff
@@ -7,21 +7,21 @@ import {
 } from '../../action-creators'
 
 import { createTestStore } from '../../test-helpers/createTestStore'
+import setCursorFirstMatch from '../../test-helpers/setCursorFirstMatch'
 
 describe('normal view', () => {
 
   it('move cursor to previous sibling', () => {
 
     const store = createTestStore()
 
-    store.dispatch([
-      importText(RANKED_ROOT, `
+    store.dispatch(importText(RANKED_ROOT, `
       - a
         - a1
-      - b`),
-      { type: 'setCursor', thoughtsRanked: [{ value: 'b', rank: 1 }] },
-      cursorPrev()
-    ])
+      - b`))
+
+    setCursorFirstMatch(['b'])(store.getState())
```

Comment:
This line won't do anything, since you're not using the return value. `setCursorFirstMatch` is a reducer, so it returns the new state.

Reply:
@pushkov-fedor